src/addons/Ref/Ref.js

```diff
@@ -24,8 +24,11 @@ export default class Ref extends Component {
 
     // Heads up! Don't move this condition, it's a short circuit that avoids run of `findDOMNode`
     // if `innerRef` isn't passed
-    // eslint-disable-next-line react/no-find-dom-node
-    if (innerRef) innerRef(findDOMNode(this))
+    try {
+      // eslint-disable-next-line react/no-find-dom-node
+      if (innerRef) innerRef(findDOMNode(this))
+      // eslint-disable-next-line no-empty
+    } catch (_) {}
```

Comment:
Silencing the error is horrible and I'm really not happy with this line. There are 2 options: either report it with `console.error` which will pollute the unit test runs, or only do a try catch in development environment (i.e. check for `process.env.NODE_ENV === 'development`). What do you think?

Reply:
@layershifter Given your updates in Stardust, what's your thought here?

Reply:
I will backport changes from https://github.com/stardust-ui/react/pull/491, this will resolve this issue fully 👍

Reply:
Thanks for addressing this issue.